src/main/webapp/app/iris/settings/iris-settings-update/iris-common-sub-settings-update/iris-common-sub-settings-update.component.ts

```
@@ -123,15 +159,28 @@
         }
     }
 
+    onCategorySelectionChange(category: string) {
+        if (!this.subSettings) {
+            return;
+        }
+        if (!this.subSettings.enabledForCategories) {
+            this.subSettings.enabledForCategories = [];
+        }
+        if (this.subSettings.enabledForCategories?.includes(category)) {
+            this.subSettings.enabledForCategories = this.subSettings.enabledForCategories!.filter((c) => c !== category);
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)